Python/symtable.c

```diff
@@ -2706,6 +2759,11 @@ symtable_visit_annotation(struct symtable *st, expr_ty annotation, void *key)
             return 0;
         }
     }
+    if (new_conditional) {
+        if (!symtable_add_def(st, &_Py_ID(__conditional_annotations__), USE, LOCATION(annotation))) {
+            return 0;
+        }
+    }
```

Comment:
Is this needed? `__conditional_annotations__` are already added above and the tests seem to pass even with this block removed.